package.json

```diff
@@ -5,23 +5,23 @@
     "prod": "NODE_ENV=production webpack --mode production"
   },
   "devDependencies": {
-    "@babel/cli": "^7.17.10",
-    "@babel/core": "^7.18.2",
-    "@babel/plugin-transform-react-jsx": "^7.17.12",
-    "@babel/plugin-transform-runtime": "^7.18.2",
-    "@babel/preset-env": "^7.18.2",
+    "@babel/cli": "^7.18.6",
+    "@babel/core": "^7.18.6",
+    "@babel/plugin-transform-react-jsx": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.18.6",
+    "@babel/preset-env": "^7.18.6",
     "@tailwindcss/forms": "^0.5.2",
     "@types/dropzone": "^5.7.4",
     "@types/feather-icons": "^4.7.0",
     "@types/history": "^5.0.0",
```

Comment:
```suggestion

```
History has now own type declaration and this is no longer needed :)

Reply:
Alright, I removed it :+1: